### DISCUSSION TITLE
Modeling of nonlinear functions

### DISCUSSION THREAD

**Discussion by allthetimezsldo** [yalmip/YALMIP#1407] [General discussion]
How to model nonlinear functions such as trigonometric functions, exponential functions, etc

**Comment by johanlofberg** [comment]
In what context? If you use a nonlinear solver you simply use those functions, if you use an exponential cone capable solver, you try to use those restricted operators, and if you use a MILP solver you use pwa approximations etc via overloaded interp1 construction
